Readme.md

```diff
@@ -3,17 +3,33 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1379)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* **[MainWindow.xaml](./CS/MainWindow.xaml) (VB: [MainWindow.xaml](./VB/MainWindow.xaml))**
-* [MainWindow.xaml.cs](./CS/MainWindow.xaml.cs) (VB: [MainWindow.xaml.vb](./VB/MainWindow.xaml.vb))
-<!-- default file list end -->
-# How to use hyperlinks in GridControl cells
+# Data Grid for WPF - How to Display Hyperlinks in Cells
 
+This example demonstrates how to display hyperlinks in [GridControl](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.GridControl) cells. To do that, bind a column to the field that contains a hyperlink in data source. Define a [DataTemplate](https://docs.microsoft.com/en-us/dotnet/api/system.windows.datatemplate) and assign it to the [CellTemplate](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.ColumnBase.CellTemplate) property. The template should include the [HyperlinkEdit](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.HyperlinkEdit) editor. Use the [HyperlinkEdit.Text]() property to specify a custom caption instead of URL.
```

Comment:
```suggestion
This example demonstrates how to display hyperlinks in [GridControl](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.GridControl) cells. To do that, bind a column to the field that contains a hyperlink in the data source. Define a [DataTemplate](https://docs.microsoft.com/en-us/dotnet/api/system.windows.datatemplate) and assign it to the [CellTemplate](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.ColumnBase.CellTemplate) property. The template should include the [HyperlinkEdit](https://docs.devexpress.com/WPF/DevExpress.Xpf.Editors.HyperlinkEdit) editor. Use the [HyperlinkEdit.Text]() property to specify a custom caption instead of a URL.
```